### PR TITLE
Sms

### PR DESCRIPTION
**Pull request by frenautvh**
i might squash all commits after code review if you want

ref: OM-36

### CODE REVIEW

File: packages/manager/layout-ovh/src/index.js [hunk 1]
```diff
@@ -17,29 +17,35 @@ import telecomUniverseComponents from '@ovh-ux/telecom-universe-components';
...
 import freefax from '@ovh-ux/manager-freefax';
+import sms from '@ovh-ux/manager-sms';
 import welcome from '@ovh-ux/manager-welcome';
 
 import routing from './manager-layout-ovh.routes';
 import sidebar from './sidebar';
 import navbar from './navbar';
 
 import 'angular-ui-bootstrap';
+import 'ng-at-internet';
```
Comment (antleblanc) [inline packages/manager/layout-ovh/src/index.js]: Could be sorted?

File: packages/manager/layout-ovh/package.json [hunk 2]
```diff
@@ -46,11 +48,13 @@
     "lodash": "^3.10.1",
     "matchmedia-ng": "^1.0.8",
     "moment": "^2.22.2",
+    "ng-at-internet": "^3.1.1",
     "ng-csv": "^0.3.6",
     "ovh-angular-actions-menu": "^4.0.0",
     "ovh-angular-apiv7": "^1.2.8",
     "ovh-angular-http": "^4.0.0-alpha.1",
     "ovh-angular-otrs": "https://github.com/ovh-ux/ovh-angular-otrs.git#17796f93cbb0a586d08dd17030e4aa2f9f47a34e",
+    "ovh-angular-pagination-front": "ovh-ux/ovh-angular-pagination-front",
```
Comment (antleblanc) [inline packages/manager/layout-ovh/package.json]: Could you specify a release number instead?

File: packages/manager/sms/package.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@ovh-ux/manager-sms",
```
Comment (antleblanc) [inline packages/manager/sms/package.json]: It would be great if you add some meta informations `description`, `keywords`:)

Comment (frenautvh) [reply packages/manager/sms/package.json]: done

**Pull request merged**